src/main/resources/application.yaml

```
@@ -12,6 +12,12 @@ datastore:
   dataset:
     dir: ${datastore.root}/dataset
 ---
+dataservice:
+  url:
```

I suggest a common root path since all endpoints will be served from the same service
```suggestion
  url: http://127.0.0.1:8000
```